src/Landing/containers/Landing.jsx

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import RecommendedArticles from '../components/RecommendArticles';
+import LatestArticles from '../components/LatestArticles';
+import fetchLatestArticles from '../redux/actions/LatestArticles';
+
+class Landing extends Component {
+  componentWillMount() {
+    this.props.fetchLatestArticles();
+  }
+
+  render() {
+    const { latestArticles, recommendedArticles } = this.props;
+    return (
+      <>
```

Comment:
What happens if you use a normal <div> instead of the fragment?

Reply:
I have switched to using <div> instead of the <>